.travis.yml

```diff
@@ -7,10 +7,14 @@ addons:
     packages:
     - nginx
 
+language: node_js
+node_js:
+  - "12.18.1"
+
 before_install:
 - gem install mustache
-
-language: bash
+- npm install
```

Comment:
Shouldn't this be npm ci?

Reply:
[Travis docs](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/) said that it would automatically use npm ci if a package-lock.json existed and the node version supported it so I just left it this way. I can change it though.

Reply:
I think we do `npm ci` everywhere else, so it feels like it should be consistent. OTOH if that's the new node pattern, then... I don't feel too strongly.